### PR TITLE
Collabora enhancements

### PR DESCRIPTION
**Pull request by enoch85**
Fix some smaller things like adding the App and better failure messages to the user. Also add some more checks.

### CODE REVIEW

File: static/collabora.sh [hunk 1]
```diff
@@ -100,6 +114,23 @@ else
...
+    echo "Removing old Docker instanses..."
+    docker stop $DOCKERPS
+    docker rm $DOCKERPS
+fi
+
+# Enable RichDocuments (Collabora App)
+if [ -d $NCPATH/apps/richdocuments ]
+then
+    sudo -u www-data php $NCPATH/occ app:disable richdocuments
+    rm $NCPATH/apps/richdocuments
```
Comment (ChristophWurst) [inline static/collabora.sh]: ``rm -r`` since it's probably a directory?

Comment (enoch85) [inline static/collabora.sh]: Thanks!

**Comment by enoch85** [comment]
@ChristophWurst Anything else?

**Comment by enoch85** [comment]
Docker check fails:

```
See 'docker inspect --help'.

Usage:	docker inspect [OPTIONS] CONTAINER|IMAGE|TASK [CONTAINER|IMAGE|TASK...]

Return low-level information on a container, image or task
collabora.sh: line 132: [: =: unary operator expected
```

**Comment by enoch85** [comment]
Tested and works! 

Left to do: request occ command for entering $SUBDOMAIN with occ.

**Comment by ChristophWurst** [comment]
> enoch85 commented a day ago
> 
> @ChristophWurst Anything else?

Now that you merged the PR anyway, I'm curious why you even care about the extra work of preparing a PR if you do not request others to review your changes? I see that other PRs were merged in the same way. Usually we do PRs for Nextcloud projects so that at least two people have seen the changes (the author and a second person). If you merge the PRs yourself, it's like you commit and push to master directly.

**Comment by enoch85** [comment]
@ChristophWurst Yeah, you're right. When @jnweiger and me first started to develop the ownCloud VM we decided to push to master directly, and since I don't know git that well I continued down that path.

Now I'm starting to learn git and just understood how to make PRs, verify your commits and such. Making PR's is very easy as you already know;

```
git checkout -b new-branch
git commit -S -a -m "commit message"
git push origin new-branch
```

Then make a PR out of that new branch.

Anyway, to answer your question. PR's are very good because others can comment and review like you just did, but it also takes time as the designated maintainers are often away or busy doing other things. In this case I wanted to push out the new fix asap as the collabora script is included in the production version and was failing atm. So I tested it myself, confirmed it to work, and then pushed it.

**TL:DR** I wanted to push this as fast as possible as there could be some risk that the collabora script was failing without these changes (checks and such). 

Hope that explains it. Thanks for your input. :)

**Comment by ChristophWurst** [comment]
I see your point, that is a valid approach I would say. I'd always prefer having PRs over direct master commits/pushes anyway. So when you do merge your own PRs without a review, it would still be nice to leave a comment why you did so. This way others are able to catch up on what has been changed in the repository recently and why there's no review 😉
